backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -42,7 +42,7 @@ trait BackendLifecycleActorFactory {
   def runtimeAttributeDefinitions(initializationDataOption: Option[BackendInitializationData]): Set[RuntimeAttributeDefinition] = Set.empty
 
   lazy val fileHashingFunction: Option[FileHashingFunction] = None
-  lazy val fileHashingWorkerCount: Int = 50
+  lazy val fileHashingActorCount: Int = 50
 
-  def fileContentsHasherActorProps: Props = FileHasherWorkerActor.props(fileHashingFunction)
+  def fileHashingActorProps: Props = FileHashingActor.props(fileHashingFunction)
```

Comment:
Info only: 
Oh I remember why `Contents` was in there. I was preparing for other types of file hash (e.g. path, companion `.md5`, etc). I approve of this change